mLRS/CommonTx/jr_pin5_interface.h

```diff
@@ -105,7 +108,8 @@ class tPin5BridgeBase
 
     // interface to the uart hardware peripheral used for the bridge, called in isr context
     void pin5_tx_start(void) { uart_tx_start(); }
-    void pin5_putc(char c) { uart_tx_putc_totxbuf(c); }
+    // This is clumsy.  Can we avoid directly manipulating txbuf?
+    void pin5_putbuf(uint8_t* buf, uint16_t len) { while (len--) {uart_tx_putc_totxbuf(*(buf++)); }}
```

Comment:
is this only for your changes, and would break e.g. STM32 targets

if so, pl figure out how to not break STM32. If not, consider doing an extra PR. This PR here may take longer.

Btw: I don't like the len-- buf++ construct, For pointers I generally like to go to a function protoype ... uint8_t* const buf ... (see a number of the recent changes)(helps the compiler to detect certain flaws)(we had at least one where it triggered :)).

Reply:
How does this break STM32?

Reply:
it doesn't ? I was thinking so since it's a change in the main h file.

if not when all good (except the len-- buf++ thing)

Reply:
Can you point me to the recent changes you reference?  Or, better, elaborate on what the problem is with the construct I've used and how specifying const can matter in this case and how you would prefer to write this function?

Reply:
the problem of the construct is that you can't write unit8_t* const buf in the function header. That's however the pattern I've went to in the recent times for the reason I've mentioned.